src/config/urls.py

```diff
@@ -21,6 +21,7 @@
     # Django-machina forum
     # https://django-machina.readthedocs.io/en/latest/getting_started.html#urls-configuration
     path("forum/", include(machina_urls)),
+    path("forum/", include("chigame.forums.urls")),
```

Comment:
Small issue of naming: should it be forums or just forum? As a user, somewhat confusing when I click on the "Forums" and then the naming isn't consistent. A very small thing, but could be of note in future.

Reply:
I agree, it should be called forums. I only kept it as that because the line before had "forum" as well. Will change.